rdbms-image/devicehive-backend/Dockerfile

```diff
@@ -18,7 +23,7 @@ RUN cd /usr/share \
 ENV MAVEN_HOME /usr/share/maven
 
 # download devicehive and build .jar
-RUN git clone -b development https://github.com/devicehive/devicehive-java-server.git /opt/devicehive && \
+RUN git clone -b master https://github.com/devicehive/devicehive-java-server.git /opt/devicehive && \
```

Comment:
This repo is often used for developers to run quickly the latest java-server version, not only release archives. So building .jars from source code gives more flexibility in choosing branches and versions. I think it needs to be discussed with the team.